Mathlib/Algebra/Hom/Equiv/Units/GroupWithZero.lean

```diff
@@ -33,9 +33,9 @@ protected def mulLeft₀ (a : G) (ha : a ≠ 0) : Perm G :=
 #align equiv.mul_left₀_symm_apply Equiv.mulLeft₀_symm_apply
 #align equiv.mul_left₀_apply Equiv.mulLeft₀_apply
 
-theorem mulLeft_bijective₀ (a : G) (ha : a ≠ 0) : Function.Bijective ((· * ·) a : G → G) :=
+theorem _root_.mulLeft_bijective₀ (a : G) (ha : a ≠ 0) : Function.Bijective ((· * ·) a : G → G) :=
```

Comment:
Should this also be `mul_left`? But maybe it's better to keep this pr focused tightly.